src/utils/outputs.ts

```diff
@@ -242,3 +242,99 @@ export const getSpeakingNames = (
   });
   return result;
 };
+
+export const getSpeakingNamesWithColors = (
+  speakingNames: SpeakingName[],
+  allColors: Record<string, string>,
+  colors: Record<string, HeisslufType[]>,
+  darkMode: boolean,
+  luminanceSteps: number[],
+  speakingNamesDefaultMapping: SpeakingName[]
+): any => {
+  let result: any = {};
+  let state;
+  const isStateName = (name: string) => {
```

Comment:
You can remove `return` here and inline the function.
Furthermore, move the function outside of the `getSpeakingNamesWithColors`, otherwise it will be created again when calling the parent function.